src/main/java/com/avispl/symphony/dal/infrastructure/management/sembient/sembient/SembientAggregatorCommunicator.java

```diff
@@ -1109,28 +1144,28 @@ private void filterByFloors(String buildingID, String buildingName, String[] flo
 				for (String floorName : floorNames) {
 					if (floor.trim().equals(floorName)) {
 						// Filter by region type
-						if (StringUtils.isNotNullOrEmpty(deviceTypeFilter)) {
-							String[] listTypeToBeFilter = deviceTypeFilter.split(SembientAggregatorConstant.COMMA);
-							for (String type : listTypeToBeFilter) {
-								retrieveDevices(buildingID, buildingName, floor, type.trim());
-							}
-						} else {
-							retrieveDevices(buildingID, buildingName, floor, null);
-						}
+//						if (StringUtils.isNotNullOrEmpty(deviceTypeFilter)) {
```

Comment:
Please remove the commented-out segments. If we need them back - we can always revert the changes :)

Reply:
I fixed it